### PR TITLE
Bug/presentation sign up

### PR DESCRIPTION
**Pull request by jacquesporveau**
https://www.pivotaltracker.com/story/show/141187241/comments/167429831

### CODE REVIEW

File: spec/presenters/presentation_presenter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe PresentationPresenter do
+  subject { described_class.new(presentation, nil) }
+
+  describe '#associable_events' do
```
Comment (qr8r) [inline spec/presenters/presentation_presenter_spec.rb:6]: This should probably also test that past events arent included

**Review by qr8r — CHANGES_REQUESTED** [review]
:+1:  other than minor test comment. Also #NamingIsHard

File: spec/presenters/presentation_presenter_spec.rb (hunk 1 above)
Comment (jacquesporveau) [reply spec/presenters/presentation_presenter_spec.rb:6]: Sounds good. I was thinking about testing that but my thought pattern was that the `.upcoming` method was already tested so I could skip that test. I'll make the changes.

Comment (qr8r) [reply spec/presenters/presentation_presenter_spec.rb:6]: That makes an assumption about the internal implementation of this method. Really I dont care if you use the upcoming scope or some other mechanism for filtering Events.

But what I care about testing is behaviour: 
- No past events are included 
- The currently associated event is present in the list

Comment (jacquesporveau) [reply spec/presenters/presentation_presenter_spec.rb:6]: Alright I've updated the spec now.

**Review by qr8r — APPROVED** [review]

**Pull request merged**